pkg/apis/kubebind/v1alpha1/helpers/serviceexport.go

```diff
@@ -90,6 +90,10 @@ func CRDToServiceExport(crd *apiextensionsv1.CustomResourceDefinition) (*kubebin
 		if !crdVersion.Served {
 			continue
 		}
+		// TODO
+		if onlyFirstServingVersion && !crdVersion.Storage {
+			continue
+		}
```

Comment:
would expect this to be a parameter to `CRDToServiceExport` and controlled through higher level means.

Reply:
this change can be removed. The intentions were the same as in #181